dotCMS/src/main/java/com/dotmarketing/util/LuceneQueryUtils.java

```diff
@@ -1,67 +1,44 @@
-import org.apache.lucene.queryparser.classic.QueryParser;
-import org.apache.lucene.search.BooleanClause;
-import org.apache.lucene.search.BooleanClause.Occur;
-import org.apache.lucene.search.BooleanQuery;
-import org.apache.lucene.search.TermQuery;
-
 
+/**
+ * Utility class in charge of dealing with Lucene Queries that require a specific formatting, validation, or addition of
+ * default parameters.
+ *
+ * @author Fabrizzio Araya
+ * @since Jun 21, 2018
+ */
 public class LuceneQueryUtils {
 
-
     /**
+     * Removes the {@code "query_"} prefix that might be included in the Lucene query.
+     *
+     * @param luceneQuery The Lucene query.
      *
-     * @param luceneQuery
-     * @return
+     * @return The Lucene query without the {@code "query_"} prefix.
      */
     private static String removeQueryPrefix(final String luceneQuery) {
-
         return (luceneQuery.startsWith("query_") ? luceneQuery.replace("query_", StringPool.BLANK)
                 : luceneQuery
         );
     }
 
-    private static Set<BooleanClause> filterQueryTerms = ImmutableSet.of(
-            new BooleanClause(new TermQuery(new Term("contentType","Host")),Occur.MUST_NOT)
-    );
-
     /**
-     * This method basically does two things. Gets rid of the 'query_' prefix and also adds an additional condition to ensure we exclude all content of type host
-     * Since acces to ContentType Host is limited.
-     * @param luceneQuery
-     * @return
-     * @throws ParseException
+     * This method basically does two things:
+     * <ol>
+     *     <li>Gets rid of the 'query_' prefix.</li>
+     *     <li>Adds an additional condition to ensure we exclude all content of type {@code host} since access to
+     *     Content Type Host is limited.</li>
+     * </ol>
+     *
+     * @param luceneQuery The Lucene query that will be treated.
+     *
+     * @return The sanitized query.
      */
-    public static String sanitizeBulkActionsQuery(final String luceneQuery) throws ParseException {
-
-        final String cleanedUpQuery = removeQueryPrefix(luceneQuery);
-        final QueryParser parser = new QueryParser(null, new WhitespaceAnalyzer());
-        final BooleanQuery query = (BooleanQuery) parser.parse(cleanedUpQuery);
-        final List<BooleanClause> clauses = query.clauses();
-        final Set<BooleanClause> clauseSet = new HashSet<>(clauses);
-        final BooleanQuery.Builder builder = new BooleanQuery.Builder();
-
-        for(final BooleanClause clause:clauses){
-            builder.add(clause);
-        }
-
-        //Do not add terms that are already part of the query.
-        for (final BooleanClause clause : filterQueryTerms) {
-              if(!clauseSet.contains(clause)){
-                 builder.add(clause);
-              }
-        }
-
-        return builder.build().toString();
+    public static String sanitizeBulkActionsQuery(final String luceneQuery) {
+        return removeQueryPrefix(luceneQuery) + " -contentType:host";
```

Comment:
Is this `contentType:host` required for every case? Or just when selecting all the items?

Reply:
So, this query is for bulk actions.  Hosts should never be included in buik actions (even if they get included in a content query), so I think this is ok.